CHANGES.md

```diff
@@ -127,6 +127,7 @@ To be released.
         `IReadOnlyList<T>` (was `IEnumerable<T>`).
      -  The type of `Transactions` property of `Block<T>` became
         `IReadOnlyList<T>` (was `IEnumerable<T>`).
+ -  Added error logging to `RocksDBStore`. [[#1277]]
```

Comment:
IMHO this shouldn't be recorded in the changelog.  You could bypass the changelog check by adding `[skip changelog]` in the commit message.